test/spec/provider/bpmn/EscalationEventDefinitionSpec.js

```diff
@@ -296,4 +301,101 @@ describe('escalation-event-properties', function() {
 
   });
 
+  describe('escalation code variable', function() {
+
+    it('should fetch escalation code variable of an escalation event definition element', inject(function(propertiesPanel, selection, elementRegistry) {
```

Comment:
naming: sometimes you call the test case `escalation code variable`, sometimes just `variable`